Formula/genders.rb

```diff
@@ -22,15 +22,27 @@ class Genders < Formula
     sha256 cellar: :any, high_sierra:    "31a726904f22c156b763a8bc95bd3db6e85b8bc0cf7d8a82d584bb8684241f6c"
   end
 
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
+  uses_from_macos "perl" => :build
+  uses_from_macos "python" => :build
+
   # Fix -flat_namespace being used on Big Sur and later.
   patch do
     url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
     sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
   end
 
   def install
+    ENV["PYTHON"] = which("python3")
     system "./configure", "--prefix=#{prefix}", "--with-java-extensions=no"
     system "make", "install"
+
+    # Move man page out of top level mandir on Linux
+    unless OS.mac?
+      mv prefix/"man/man3/Libgenders.3", man/"man3"
```

Comment:
I don't quite follow what this does. It looks like it doesn't do anything?

Reply:
I probably need a better comment here.  The issue is that on Linux the file `Libgenders.3` is incorrectly installed to `prefix/"man/man3"`.  This just moves the file out of that folder into `man/"man3"` and then deletes the top level `man` dir that `brew audit` doesn't allow.

Reply:
Ah yes, I forgot that `man` is in `share`. Doesn't setting this using `configure` flags work?

Reply:
Looks like common Perl manpage issue, where we often need to call `perl Makefile.PL ...` with `INSTALLSITEMAN3DIR` to override.

In this case, the Perl bindings are built internally during `make install`. The only args used are https://github.com/chaos/genders/blob/genders-1-27-3/src/extensions/perl/Libgenders/Makefile.am#L20-L27
```make
MAKEMAKERFLAGS = INSTALLDIRS=$(PERL_ARCH_INSTALL) \
		 DESTDIR=$(PERL_DESTDIR) \
		 PREFIX=$(prefix) \
		 INC=$(PERL_INC) \
		 LIBS=$(PERL_LIBS) \
		 GENDERS_LIBDIR=$(libdir) \
		 MAKEFILE=Makefile.xs \
		 FIRST_MAKEFILE=Makefile.xs
```

Reply:
Is this something we should try to fix in the Makefile?  Or is this workaround acceptable (with a better comment)?  I can definitely confirm that setting `--mandir` with `configure` has no effect, probably for the reason @cho-m posted above.

Reply:
Ok, that's fine. Let's just simplify this `unless` block. See below.